cmd/podman/kube/play.go

```diff
@@ -143,6 +145,9 @@ func playFlags(cmd *cobra.Command) {
 	replaceFlagName := "replace"
 	flags.BoolVar(&playOptions.Replace, replaceFlagName, false, "Delete and recreate pods defined in the YAML file")
 
+	detachFlagName := "detach"
+	flags.BoolVar(&playOptions.Detach, detachFlagName, true, "Return after having started the pods")
```

Comment:
Add short option "d" to be consistent with podman run -d (--detach)

How about:

flags.BoolVarP(&playOptions.Detach, detachFlagName, "d", true, "Run pods and containers in background")